src/frontend/src/test-e2e/views.ts

```diff
@@ -803,14 +802,11 @@ export class DemoAppView extends View {
     await waitToClose(this.browser);
     // wait for the demo app to update the principal
     await this.browser.waitUntil(
-      async () => (await this.getPrincipal()) !== Principal.anonymous().toText()
+      async () => (await this.getPrincipal()) !== ""
```

Comment:
that's correct?

Reply:
Yep! this used to default to the anonymous principal; now we just don't set one